main.go

```diff
@@ -118,8 +118,7 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	upath := r.URL.Path[1:]
-	fpath := filepath.Join(".", filepath.Base(upath))
+	fpath := filepath.Join(".", r.URL.Path)
```

Comment:
We should fix the other logics with `fpath`. For example, HTML was served when fpath was `index.html`, but this PR changes the logic.

Reply:
👍 I'll get to it.

Reply:
So probably rewriting

```
switch filepath.Base(fpath) {
```

to

```
switch filepath.Base(filepath.Join(".", filepath.Base(r.URL.Path[1:]))) {
```

should work by preserving the original behavior. I might miss other things.